qiita_pet/supervisor_example.conf

```diff
@@ -7,22 +7,22 @@ loglevel=debug               ; log level; default info; others: debug,warn,trace
 pidfile=/tmp/supervisord.pid ; supervisord pidfile; default supervisord.pid
 
 [include]
-files=/home/travis/miniconda3/envs/qiita
+files=/home/travis/miniconda3/envs/qiita/bin
 
 [group:qiita_all]
 programs=qiita_workers_0,qiita_workers_1,qiita_master  ; each refers to 'x' in [program:x] definitions
 
 [program:qiita_master]
-command=scripts/qiita pet webserver start --port 21174 --master
+command=qiita pet webserver --no-build-docs start --port 21174 --master
```

Comment:
when this is deployed, we'll need it to build docs for the online documentation, right?